src/extension/js/fix-pass-layout.js

```diff
@@ -336,7 +336,7 @@ export default class PassLayoutFixer {
         staff.parent().next().after(backgroundColors)
         backgroundColors.wrap('<li class="clearfix">')
 
-        const { checkboxWrapper, editTrigger } = this.createCheckboxOptionWithNumber(options, 'Highlight sections with less than', 'open spot', 'open spots',  'highlightAlmostClosedSections', 'highlightAlmostClosedSectionsThreshold', ALMOST_CLOSED_SECTIONS_MIN_VALUE, ALMOST_CLOSED_SECTIONS_MAX_VALUE)
+        const { checkboxWrapper, editTrigger } = this.createCheckboxOptionWithNumber(options, 'Highlight sections with fewer than', 'open spot ', 'open spots ',  'highlightAlmostClosedSections', 'highlightAlmostClosedSectionsThreshold', ALMOST_CLOSED_SECTIONS_MIN_VALUE, ALMOST_CLOSED_SECTIONS_MAX_VALUE)
```

Comment:
What's the reason for adding a space after "spot"/"spots"? It doesn't look to me like there are any spacing issues.

Reply:
I noticed that there was no spacing right after "spots" when the window width is >= 1578px.

<img width="345" alt="screen shot 2017-11-01 at 9 53 23 pm" src="https://user-images.githubusercontent.com/11605206/32310402-2f89502c-bf4f-11e7-97f3-08a152a43a3c.png">

Reply:
Ah, I'm testing on a laptop so the options weren't as wide for me - never ran into that case. That's an issue that should be fixed with css as I intended for the edit button to always be below the checkbox label. If you'd like to take a crack at that, feel free to do so, otherwise please revert the spaces and I'll approve the grammar fix.